src/integration.test.ts

```diff
@@ -406,4 +422,29 @@ describe(`Integration test for portal ${portal}`, () => {
       expect(data).toEqual(expectedData);
     });
   });
+
+  // TODO: Doesn't work in Node. See
+  // https://github.com/tus/tus-js-client/issues/252#issuecomment-821181951
+  // describe("Large upload integration tests", () => {
+  //   // TODO: Remove.
+  //   const client = new SkynetClient("https://siasky.xyz");
+  //   const fileData = "testing";
+  //   const filename = "name";
+  //   const file = new File([fileData], filename);
+
+  //   it("Should be able to upload and download a small file uploaded using the large upload method", async () => {
+  //     // Upload the data using the large file upload.
+
+  //     console.log("calling uploadlargefile...");
+  //     // @ts-ignore
+  //     const { skylink } = await client.uploadLargeFile(file);
+  //     console.log("finished calling uploadlargefile...");
+  //     expect(skylink).not.toEqual("");
+
+  //     // Get file metadata and check filename.
+
+  //     const { metadata } = await client.getMetadata(skylink);
+  //     console.log(metadata);
+  //   });
+  // });
```

Comment:
Let's not add commented out code. We have large chunks of dead code which is not very helpful. We can always spin this up a F/U to add the tests that we can't add right now.

Reply:
I'm hoping to get this resolved before the PR goes live. Basically, it does not seem possible to call the tus `Upload` function from Node, so the test currently errors out:

https://github.com/tus/tus-js-client/issues/252#issuecomment-821181951